pkg/adsc/adsc.go

```diff
@@ -489,11 +489,11 @@ func (a *ADSC) handleRecv() {
 		if err != nil {
 			a.RecvWg.Done()
 			adscLog.Infof("Connection closed for node %v with err: %v", a.nodeID, err)
-			a.errChan <- err
```

Comment:
I donot think this is the right fix,.

Suggest 
```
			select {
			case a.errChan <- err:
			default:
			}
```

Reply:
good suggestion, thinks.